.github/workflows/push.yml

```
@@ -54,11 +54,14 @@ jobs:
         uses: splunk-soar-connectors/.github/.github/actions/integration-tests@main
   
   security-scans:
-    runs-on: ubuntu-latest
+    runs-on: 
+      - codebuild-integration-tests-${{ github.run_id }}-${{ github.run_attempt }}
+      - image:custom-linux-875003031410.dkr.ecr.us-west-2.amazonaws.com/soar-connectors/pytest:f7150dbb7f347d35f8f4bb285d36985ecd4cf231
     needs: [build, compile]
+
     steps:
-      - name: Setup Environment
-        uses: splunk-soar-connectors/.github/.github/actions/env-setup@main
+      - name: Check out actions repo
+        uses: actions/checkout@v4
 
       - name: Run Security Scans
-        uses: splunk-soar-connectors/.github/.github/actions/security-scans@main
+        uses: splunk-soar-connectors/.github/.github/actions/security-scans@mnordby/PAPP-35405
```

Probably change this before merge
```suggestion
        uses: splunk-soar-connectors/.github/.github/actions/security-scans@main
```

---

Yup